server/models/StageModel.js

```diff
@@ -24,8 +20,9 @@ const stageSchema = new mongoose.Schema({
   stageLongitude: Number,
   stageLatitude: Number,
   locationName: String,
```

Comment:
Just adding a suggestion, what do you think if we further clarified the following attributes for Stage/Trip Models:
 
StageModel.js
- locationName -- change to --> stageLocation
- stageLongitude
- stageLatitude

TripModel.js
- destination -- change to --> tripLocation
- tripLongitude
- tripLatitude